demos/quick-start/README.md

```diff
@@ -27,23 +27,23 @@ can try querying some data, but you don't have the credentials required to
 connect:
 ```
 psql \
-  --host localhost \
-  --port 5432 \
-  --set=sslmode=disable \
-  --username secretless \
-  -d quickstart \
+  "host=localhost
```

Comment:
Note here in a comment why you're using this "universal" syntax, so some future dev doesn't inadvertently change it.